mastadonApp/build.gradle.kts

```diff
@@ -30,23 +33,18 @@ android {
   }
   buildTypes {
     getByName("release") {
-      isMinifyEnabled = true
+      signingConfig = signingConfigs.getByName("debug")
```

Comment:
We probably don't want to keep it like this right? Maybe a TODO for when it comes time to actually making release builds for the play store?